### PR TITLE
(2288) Show enforcement bodies and qualifying bodies on professions page

### PR DESCRIPTION
**Pull request by pezholio**
This adds enforcement bodies and qualifying bodies on the professions show pages. Enforcement bodies are shown in their own new section, and Awarding Bodies are shown in the Qualifications and Experience section

## Screenshot 

image

### CODE REVIEW

File: src/qualifications/presenters/qualification.presenter.spec.ts [hunk 1]
```diff
@@ -467,6 +468,40 @@ describe(QualificationPresenter, () => {
...
+            qualification,
+            createMockI18nService(),
+            organisations,
+          );
+
+          const summaryList = await presenter.summaryList(true, false);
+          const awardingBodyRole = summaryList.overviewSummaryList.rows.find(
+            (r) =>
+              r.key['text'] ===
+              'Translation of `professions.show.qualification.awardingBodies`',
```
Comment (bryangaledxw) [inline src/qualifications/presenters/qualification.presenter.spec.ts:490]: Suggest use `translationOf`

File: src/qualifications/presenters/qualification.presenter.ts [hunk 2]
```diff
@@ -79,6 +81,14 @@ export default class QualificationPresenter {
       );
     }
 
+    if (this.awardingBodies && this.awardingBodies.length) {
```
Comment (bryangaledxw) [inline src/qualifications/presenters/qualification.presenter.ts:84]: Consider `if (this. awardingBodies?.length)`

File: src/professions/presenters/organisation-list.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Organisation } from '../../organisations/organisation.entity';
```
Comment (bryangaledxw) [inline src/professions/presenters/organisation-list.ts:1]: Should this be used in the Qualifications presenter?

File: views/professions/show.njk.spec.ts [hunk 4]
```diff
@@ -98,4 +98,44 @@ describe('show.njk', () => {
       },
     );
   });
+
```
Comment (bryangaledxw) [inline views/professions/show.njk.spec.ts:101]: We have Nunjucks tests??!

**Review by bryangaledxw — APPROVED** [review]
LGTM

File: views/professions/show.njk.spec.ts (hunk 4 above)
Comment (pezholio) [reply views/professions/show.njk.spec.ts:101]: Yeah, I added them in the last feature. We're getting a lot of different combinations of organisation types, so I didn't want to add yet more seeds to the tests, and yet more slow feature tests.

**Pull request merged**